sources/packages/backend/libs/integrations/src/services/ssh/ssh.service.ts

```diff
@@ -29,4 +29,14 @@ export class SshService {
       // Source: https://www.npmjs.com/package/ssh2-sftp-client#sec-5-2-21
     }
   }
+
+  /**
+   * Check if the provided error is an SSH error with the given error code.
+   * @param error SSH error to be checked.
+   * @param errorCode Error code to be checked.
+   * @returns True if the error is an SSH error with the given code, false otherwise.
+   */
+  static hasError(error: unknown, errorCode: SSHErrorCodes): boolean {
```

Comment:
name could be hasSSHError

Reply:
It can, but it is a static method, inside a service called `SshService`, that will be consumed as `SshService.hasError`. Does it really require `SSH`? If yes, then the `createClient` should be renamed to `createSSHClient` 😉